src/_locales/en_US/messages.json

```diff
@@ -129,6 +129,10 @@
         "message": "Prevent WebRTC from leaking local IP address",
         "description": "Checkbox label on the general settings page"
     },
+    "options_x_client_data_setting": {
+        "message": "Remove \"x-client-data\" header from outgoing requests",
```

Comment:
We should phrase this in a way that's intelligible to non-technical people. Take a look at our existing privacy overrides. Something like `Disable sending [NON TECHNICAL DESCRIPTION OF VARIATIONS HEADER DATA] to Google ("X-Client-Data header")`.

And, yes, exactly, we should also have a "learn more" icon that links to somewhere helpful.